static/app/views/starfish/views/appStartup/screenSummary/eventSamples.tsx

```diff
@@ -49,10 +55,9 @@ export function EventSamples({
     'OR',
     'span.description:"Warm Start"',
     ')',
+    `${SpanMetricsField.APP_START_TYPE}:${startType || '[cold,warm]'}`,
```

Comment:
feels like `cold` and `warm` can be stored and reused as constants, like `STARTUP_SPANS`. Especially since this comes up again in `spanOperationTable`